pkg/assert/validate.go

```diff
@@ -33,6 +33,7 @@ func validate(exec func(string) (string, error), args ...string) error {
 			if err != nil {
 				shared.LogLevel("error", "error from runAssertion():\n %s\n", err)
 				close(errorsChan)
+
```

Comment:
This one shouldn't need a new line since it should fall within the block-size of 4

Reply:
block size of 4 only counts form the beginning of the func, this is to avoid needing extra line in too small funcs or just returning funcs

Reply:
oohh interesting! I'm fine with this then!I also checked the tests on the repo (https://github.com/ssgreg/nlreturn/blob/master/testdata/src/p/p.go) and it looks like we can still have hings bunched at least for the one-line error returns, like:
```go
// some code

_, err := someFunc()
if err != nil {
  return fmt.Sprintf("There was an error: %v", err)
}

// some more code
```

So that's nice and clear!

Reply:
agreed 
yep that was the idea